src/intl/en/page-what-is-ethereum.json

```
@@ -61,26 +58,24 @@
   "page-what-is-ethereum-slide-4-desc-1": "Play to earn games (where players are actually rewarded for playing the games) have recently emerged and are transforming the gaming industry. Traditionally, it is often prohibited to trade or transfer in-game assets to other players for real money. This forces players to use black market websites that are often a security risk. Blockchain gaming embraces the in-game economy and promotes such behavior in a trusted manner.",
   "page-what-is-ethereum-slide-4-desc-2": "Moreover, players are incentivized by being able to trade in-game tokens for real money and thus being truly rewarded for their play time.",
   "page-what-is-ethereum-meet-ether-title": "Meet ether, Ethereum's cryptocurrency",
-  "page-what-is-ethereum-meet-ether-desc-1": "Ethereum has a native cryptocurrency called ether (ETH). It is purely digital, and you can send it to anyone anywhere in the world instantly. The supply of ETH isn’t controlled by any government or company - it is decentralized and completely transparent. New coins (also commonly called tokens) are issued only to stakers who secure the network.",
-  "page-what-is-ethereum-meet-ether-desc-2": "Every action on the Ethereum network requires a certain amount of computational power. This fee is paid in the form of ether. This means you need at least a small amount of ETH to use the network.",
+  "page-what-is-ethereum-meet-ether-desc-1": "Many actions on the Ethereum network require some work to be done on Ethereum's embedded computer (known as the EVM). This computation is not free; it is paid for using Ethereum's native cryptocurrency called ether (ETH). This means you need at least a small amount of ether to use the network.",
```

If we're going to mention the EVM, I would spell it out. We don't mention it anywhere else on the page.
```suggestion
  "page-what-is-ethereum-meet-ether-desc-1": "Many actions on the Ethereum network require some work to be done on Ethereum's embedded computer (known as the Ethereum Virtual Machine). This computation is not free; it is paid for using Ethereum's native cryptocurrency called ether (ETH). This means you need at least a small amount of ether to use the network.",
```